conf/globals.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * @fileoverview Globals for ecmaVersion/sourceType
+ * @author Nicholas C. Zakas
+ */
+
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Globals
+//-----------------------------------------------------------------------------
+
+const commonjs = {
+    require: false,
+    exports: false,
+    module: false
+};
```

Comment:
There are two differences between this and [globals.commonjs](https://github.com/sindresorhus/globals/blob/332404584ae6721500cfcddb034a3fae26618666/globals.json#L1220). Is it intentional?

* `global` is missing here.
* `exports` is readonly here.

`global` was added to `globals.commonjs` per https://github.com/eslint/eslint/issues/4728.

Assignment to `exports` is usually a bug, but there's [this style](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/exports-style.md#allowbatchassign).

Reply:
Not intentional. I'll update it. Thanks.